src/components/faq-page/GeneralInfoPage.tsx

```diff
@@ -0,0 +1,390 @@
+              For DOST and CHED grants: Eligible. However, GIA/JSF grant will be
+              adjusted if the total scholarship grant of the scholar exceeds the
+              student’s total assessment.
+            </li>
+            <li className="mt-5">
+              STEP/EBSU: Not eligible for a GIA scholarship, but still eligible
+              for the JSF grant.
+            </li>
+          </ol>
+        </div>
+
+        {/* Grades to maintain in order renew the scholarship */}
+        <p className="mb-5 mt-5">
+          c. Grades to maintain in order renew the scholarship
+        </p>
+        <div className="ml-8">
+          <ol className="list-[lower-roman] pl-6 mt-5">
+            <li className="mt-5">For JSF Grantees: A QPI of 3 (B)</li>
+            <li className="mt-5">For GIA Grantees: A QPI of 2.5 (C+)</li>
+          </ol>
+        </div>
+      </div>
+
+      <h2 className="text-blue font-bold text-main mt-12">
+        5. Graduation Honors
+      </h2>
+      <div className="ml-8 text-blue">
+        <p className="mb-5 mt-5">
+          a. Scholastic requirements for academic honors for graduation are as
+          follows:
+        </p>
+        <div className="ml-8">
+          <ol className="list-[lower-roman] pl-6 mt-5">
+            <li className="mt-5">
+              Summa cum laude - cumulative QPI of 3.80 to 4.00
+            </li>
+            <li className="mt-5">
+              Magna cum laude - cumulative QPI of 3.60 to 3.79
+            </li>
+            <li className="mt-5">Cum laude - cumulative QPI of 3.40 to 3.59</li>
+          </ol>
+        </div>
+      </div>
+
+      <h2 className="text-blue font-bold text-main mt-12">
+        6. The University Student Government and Organizations
+      </h2>
+      <div className="ml-8 text-blue mb-48">
+        <p className="mb-5 mt-5">
+          <span className="font-bold">
+            a. SAMAHAN NG MAG-AARAL NG PAMATASANG ATENEO DE DAVAO (SAMAHAN)
+          </span>{' '}
+          - the official student government of the Ateneo de Davao University.
+        </p>
+        <p className="mb-5 mt-5">
+          <span className="font-bold">
+            b. COUNCIL OF ORGANIZATIONS OF THE ATENEO - DAVAO (COAD)
+          </span>{' '}
+          - the umbrella organization of all recognized clubs in the Ateneo de
+          Davao University.
+        </p>
+        <p className="mb-5 mt-5">
+          <span className="font-bold">
+            c. COUNCIL OF CLASS PRESIDENTS (CCP)
+          </span>{' '}
+          - is composed of all class presidents (from the first to fifth years)
+          elected for the current academic year.
+        </p>
+        <p className="mb-5 mt-5">
+          <span className="font-bold">d. COMELEC</span> - the constitutional
+          electoral and independent body of Ateneo de Davao University.
+        </p>
+      </div>
+    </div>
+  );
+};
+export default GeneralInfoPage;
+
+const WPA = () => {
```

Comment:
Separate WPA, QPI, Allowable Absences in a new file

Reply:
May own file sila each I mean

Reply:
You can also do const `const WPA = ( .... )` instead of `const WPA = { return (....) }` Do this for QPI and Absences also